src/json-observe.js

```diff
@@ -1,5 +1,12 @@
-function _isArray(obj) { return Array.isArray ? Array.isArray(obj) : function (obj) { return obj.push && typeof obj.length === 'number'; }; }
-;
+var _isArray;
+if (Array.isArray) {
```

Comment:
Personally, I'd use `var _isArray = Array.isArray || function...`, but it's just a preference